grails-app/migrations/0.8.x/changelog-2022-06-08-1200-add-product-merge-role.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+
+  <changeSet author="DariuszRodzewicz" id="0806202212000-0">
+    <insert tableName="role">
+      <column name="id" value="6"/>
```

Comment:
I wasn't too sure about the `id` some of the roles in the DB are integers like `ROLE_ASSISTANT: 4: `, `ROLE_ADMIN: 1`, `ROLE_BROWSER: 3` and some are hashed like `ROLE_APPROVER: 09f5e1167751da090177634f04a42b66`

Reply:
That's because there was no migration for `ROLE_APPROVER`, could you add it too (with a check to add it if it does not exist in the DB)?